cloud/wfm-rest-api/src/ApiConfig.ts

```diff
@@ -0,0 +1,31 @@
+
+/**
+ * Module configuration interface
+ * Holds all values that can be changed to modify module behavior
+ */
+export interface ApiConfig {
+  /** Used to create workorder route */
+  workorderApiName?: string;
+  /** Used to create workflow route */
+  workflowApiName?: string;
+  /** Used to create result route */
+  resultApiName?: string;
```

Comment:
I think we can get rid of the `Used to`s here, and more detail for the client developer: `Root path for the Result http endpoints.`